src/components/Navbar.js

```diff
@@ -1,15 +1,8 @@
 import "./Navbar.css";
-import { useState, useEffect } from "react";
 
 const Navbar = () => {
-  const [display, setDisplay] = useState("none");
-
-  const toggleDropdown = () => {
-    if (display === "none") {
-      setDisplay("flex");
-    } else {
-      setDisplay("none");
-    }
+  const toggleDropdown = (event) => {
+    event.currentTarget.classList.toggle("is-open");
```

Comment:
Reacti maailmas on parem praktika ikkagi selliseid muudatusi state-is hoida, mitte otse DOM-i elemendi classListi muutma minna.

Ma lahendaks selle nii, et sul on state nimega "isNavMenuOpen" ja nupuvajutusega toggled seda. Seejärel vastavalt state-ile otsustad, kas on CSS-i jaoks lisaklass küljes või mitte. Laialt levinud library selle tegevuse mugavdamiseks on see: https://www.npmjs.com/package/classnames <- proovi sellega lahendada. :)